requirements-dev.txt

```diff
@@ -18,6 +18,7 @@ matplotlib==3.1.2
 mock==1.3.0
 monty==3.0.4
 mpmath==1.1.0
+munch==2.5.0
```

Comment:
We shouldn't need `bunch` then, right, so we can clean it up?

Reply:
yes, I can do that.